views/layout.erb

```diff
@@ -0,0 +1,34 @@
+<body>
+
```

Comment:
I had no idea how to sign in so I couldn't get to the point of seeing my username displayed. So I won't mark this section of the test.